src/genegraph/transform/clinvar/clinical_assertion.clj

```diff
@@ -128,32 +128,32 @@
         add-contextualized)))
 
 
-(defn get-trait-by-id [trait-id release-date]
-  (let [;; Most recent trait with trait-id no later than release-date
-        rs (q/select "select ?i where {
+#_(defn get-trait-by-id [trait-id release-date]
+    (let [;; Most recent trait with trait-id no later than release-date
+          rs (q/select "select ?i where {
                       { ?i a :vrs/Disease } union { ?i a :vrs/Phenotype }
                       ?i :cg/clinvar-trait-id ?trait_id .
                       ?i :cg/release-date ?release_date .
                       FILTER(?release_date <= ?max_release_date)
                       }
                       order by desc(?release_date)
                       limit 1"
-                     {:trait_id trait-id
-                      :max_release_date release-date})]
-    (when (= 0 (count rs))
-      (log/error :fn :get-trait-by-id
-                 :msg "No matching trait found"
-                 :trait-id trait-id
-                 :release-date release-date))
-    (first rs)))
+                       {:trait_id trait-id
+                        :max_release_date release-date})]
+      (when (= 0 (count rs))
+        (log/error :fn :get-trait-by-id
+                   :msg "No matching trait found"
+                   :trait-id trait-id
+                   :release-date release-date))
+      (first rs)))
```

Comment:
Delete it.

Reply:
Think this is a temporary comment due to issues with the generation of a unique sha that Larry/Kyle spoke about Friday?

Reply:
This was originally used to resolve the foreign key from trait-set to trait when the trait-set is ingested. I commented it out because it's no longer used after changing the code to only do that foreign key traversal when the trait-set needs to be serialized for output, using `get-trait-resource-by-version-of` instead. I will just remove this and similarly the `get-trait-set-by-id`. It's simple enough to re-create it if needed later.